x/sequencer/keeper/params.go

```diff
@@ -3,42 +3,70 @@ package keeper
 import (
 	"time"
 
+	errorsmod "cosmossdk.io/errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	"github.com/dymensionxyz/dymension/v3/x/sequencer/types"
+	"github.com/dymensionxyz/gerr-cosmos/gerrc"
 )
 
-// GetParams get all parameters as types.Params
-func (k Keeper) GetParams(ctx sdk.Context) types.Params {
-	return types.NewParams(
-		k.MinBond(ctx),
-		k.UnbondingTime(ctx),
-		k.NoticePeriod(ctx),
-		k.LivenessSlashMultiplier(ctx),
-	)
+const (
+	HubExpectedTimePerBlock = 6 * time.Second
```

Comment:
this is not going to be very maintainable, and it could potentially be inaccurate. I would suggest this to be calculated and cached

Reply:
it's fine for now. used for param change validation